savant/deepstream/drawfunc.py

```diff
@@ -113,20 +113,20 @@ def _draw_bounding_box(
         else:
             padding = (0, 0, 0, 0)
         artist.add_bbox(
-            bbox=obj_meta.bbox,
-            border_color=spec.border_color.rgba,
-            border_width=spec.thickness,
-            bg_color=spec.background_color.rgba,
-            padding=padding,
+            obj_meta.bbox,
```

Comment:
Positional args are supposed to be more performant https://peps.python.org/pep-0570/#performance

When per-object functions are concerned it might make a difference.

Readability disadvantage seems to be minor and largely mitigated by IntelliSense-like IDE features.